src/go/rdctl/cmd/root.go

```diff
@@ -77,16 +82,30 @@ func init() {
 	rootCmd.PersistentFlags().StringVar(&password, "password", "", "overrides the password setting in the config file")
 }
 
-func doRequest(method string, command string) ([]byte, error) {
+func versionCommand(version string, command string) string {
+	if version == "" {
+		return fmt.Sprintf("%s/%s", apiVersion, command)
+	}
+	return fmt.Sprintf("%s/%s", version, command)
+}
+
+func makeURL(host string, port string, command string) string {
+	if command[0] == '/' {
+		return fmt.Sprintf("http://%s:%s%s", host, port, command)
```

Comment:
Consider using [`net.JoinHostPort`](https://pkg.go.dev/net#JoinHostPort).

Reply:
What's the gain of bringing in another function call?

Reply:
Just that it handles more corner cases (like IPv6 addresses). But now that we've _considered_ it, it's perfectly alright to say it's not worth it :D

Reply:
Not worth it now.